.dockerignore

```diff
@@ -0,0 +1,36 @@
+/.git
+/.dockerignore
+.gitignore
+Dockerfile
+docker-compose.yml
+docker-compose.*.yml
+
+/.project
+/.pydevproject
+/.settings
+/.idea
+/.sonar
+/.venv
+/.env
+/.env.local
+/.env.local.example
+/.env.production
+/.env.production.example
+/.vscode
+/env
+/staticfiles
+/media
+
+Thumbs.db
+.DS_Store
+*~
+
+*.prefs
+*.pyc
+*.orig
+*.db
+*.sqlite
+*.sqlite3
+
+local_settings.py
+.vscode
```

Comment:
Please add new line at the end of file ( we will have to add flake8 configs to this project as well )